fontforgeexe/charview.c

```diff
@@ -3933,7 +3933,7 @@ static void CVCharUp(CharView *cv, GEvent *event ) {
 	    TRACE("was on charselector\n");
 	    GWidgetIndicateFocusGadget( cv->hsb );
 	}
-	else
+	else if ( cv->charselector != NULL )
```

Comment:
So there was a crash when cv->charselector was NULL? Why would cv->charselector be NULL in this context?